modules/mysql/metadata.display.yaml

```diff
@@ -43,6 +43,9 @@ spec:
         backup_configuration:
           name: backup_configuration
           title: Backup Configuration
+          alt_defaults:
+            - type: ALTERNATE_TYPE_UNSPECIFIED
```

Comment:
I think we want these to be `ALTERNATE_TYPE_DC` but will let @q2w confirm

Reply:
@qz267 Let's use `ALTERNATE_TYPE_DC ` which we have introduced for ADC!

Reply:
DONE